### DISCUSSION TITLE
Internal package causing problems

### DISCUSSION THREAD

**Discussion by lowell108** [elimity-com/scim#136] [Package Questions] [answered]
This is a great Go piece of code, but I can't get past the "internal package not allowed" problem. 

I just want to use your great code as a skeleton for a SCIM server I need to write. Can you refactor to not use Internal?

**Comment by q-uint** [comment, marked as answer]
Are you trying to import the `internal` package into you implementation? You can read more about internal packages [here](https://go.dev/doc/go1.4#internalpackages).

**Comment by lowell108** [comment]
Thank you!  I worked it out. 

Also, your code is beautiful. I'm going to add a main class and run it on a server/hook it up to my tenant!
Thanks for sharing it!
Lowell Gilbertson lowellgilbertson dot com 

    On Tuesday, December 7, 2021, 02:44:16 AM EST, Quint Daenen ***@***.***> wrote:  
 
 


You you trying to import the internal package into you implementation? You can read more about internal packages here.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
